hicexplorer/test/test_hicmatrix.py

```diff
@@ -77,12 +80,12 @@ def test_convert_to_zscore_matrix():
     # make Hi-C matrix based on test matrix
     hic = hm.hiCMatrix()
     hic.matrix = csr_matrix(mat)
-    cut_intervals = [('chr', idx, idx + 10, 0) for idx in range(0, mat.shape[0] * 10, 10)]
+    cut_intervals = [('chr', idx, idx + 10, 0)
+                     for idx in range(0, mat.shape[0] * 10, 10)]
```

Comment:
Can you write it one line? It should be possible to deactivate the linting rule here.